content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+
+We first initialize DVC inside the project to create an experiment.
+
+```dvc
+$ dvc init
+```
+
+Then we add the dataset to the project:
+
+```dvc
+$ dvc add data/images
+```
+
+This creates a `data/images.dvc` file that contains all the relevant metadata of
+the directory. You can add and commit `data/images.dvc` file, and changes in
+`.gitignore` that hides the `data/images/` directory from Git.
+
+```dvc
+$ git add data/.gitignore data/images.dvc
+$ git commit -m "Dataset added"
+```
+
+DVC experiments are run by specifying their commands, outputs, parameters and
+dependencies. We specify an experiment command by `dvc stage add`.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+DVC creates `dvc.yaml` file and modifies `.gitignore` for Git to ignore certain
+artifacts.
+
+```dvc
+$ git add dvc.yaml .gitignore
+$ git commit -m "added data and the experiment"
+```
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default hyperparameter values requires only the
+command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-7683f
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+It runs the pipeline starting from the basic dependencies and produces
+`metrics.json` file for the default state.
+
+<details>
+
+### If you used `dvc repro` before
+
+Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
+have a DVC project, you may already be using `dvc repro`.
+
+In DVC 2.0 `dvc exp run` supersedes `dvc repro`. Both of these commands run the
+pipeline.
+
+We use `dvc repro` to run the pipeline as found in the <abbr>workspace</abbr>.
+All the parameters and dependencies are retrieved from the current workspace. It
+doesn't use any special objects to track the experiments. When you have large
+number of experiments that you don't want to commit all to Git, it's better to
+use `dvc exp run`. It allows to change the parameters quickly, can track the
+history of artifacts and has facilities to compare these experiments easily.
+
+`dvc repro` is still available to run the pipeline when these extra features are
+not needed.
```

Comment:
Not clear what we're trying to say about `repro`. Can probably be a shorter ~~note~~ paragraph 🙂

Reply:
I'm trying to say that `∂vc exp run` is `repro` + features 🤭😄

Reply:
(Same answer 😬)